docs/src/api/request_collection/chain_source.md

```diff
@@ -81,6 +82,24 @@ recipe: login
 trigger: !always
 ```
 
+### Chain Request Section
+
+This defines which section of the response (headers or body) should be used to load the value from.
+
+| Variant      | Type       | Description                                                                                                                  |
+| ------------ | ---------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `body`       | None       | The body of the response                                                                                                     |
+| `header`     | `String`   | A specific header from the response. If the header appears multiple times in the response, only the first value will be used |
```

Comment:
String should be lowercase
```suggestion
| `header`     | `string`   | A specific header from the response. If the header appears multiple times in the response, only the first value will be used |
```

Reply:
Added to changelog and changed to `string` here.

Thanks for the walking through of this MR, hopefully my next contribution will be cleaner!